src/agentscope/mcp/_stateful_client_base.py

```diff
@@ -83,6 +90,7 @@ async def close(self) -> None:
             logger.info("MCP client closed.")
         except Exception as e:
             logger.warning("Error during MCP client cleanup: %s", e)
+            raise e
```

Comment:
The `except` statement seems meaningless if we raise the error here. 

So if we decide to raise it, perhaps we should remove the try except statements.